src/govuk/components/character-count/template.njk

```diff
@@ -29,9 +29,6 @@
   {{ govukHint({
     text: 'You can enter up to ' + (params.maxlength or params.maxwords) + (' words' if params.maxwords else ' characters'),
     id: params.id + '-info',
-    classes: 'govuk-character-count__message' + (' ' + params.countMessage.classes if params.countMessage.classes),
-    attributes: {
-      'aria-live': 'polite'
-    }
+    classes: 'govuk-character-count__message' + (' ' + params.countMessage.classes if params.countMessage.classes)
```

Comment:
@querkmachine Do you know what would happen if someone updated to the latest version of govuk-frontend with this change, but failed to update their HTML (assuming they're using HTML, not Nunjucks)? Following on from what David said in the planning meeting, I'm wondering if this change makes it a breaking change.

Reply:
My assumption has been that nothing will happen and the attribute is just redundant, as the inner text of this element is never updated, and thus `aria-live` is never 'activated'. Happy to do a quick whip around to test that assumption, though.

Reply:
Having quickly ran through JAWS/NVDA/VoiceOver in a couple of browsers each, there doesn't seem to be any side effect to the `aria-live` attribute being present.

As the attribute was previously included in HTML even if the character count was never initialised, I don't think that it still being present in an upgrading service represents a breaking change and is more akin to a recommended change (as `tabindex` was for error summaries in the last release).

Reply:
Brill, thanks for checking that! 👍